docs/ui/preferences/virtual-machine/emulation.md

```diff
@@ -7,23 +7,22 @@ title: Emulation (macOS)
   <link rel="canonical" href="https://docs.rancherdesktop.io/ui/preferences/virtual-machine/emulation"/>
 </head>
 
- ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation.png)
-
 ### QEMU
 
-The "[QEMU](https://www.qemu.org/documentation/)" option is enabled by default and is used to run a guest operating system.
+ ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation.png)
 
- ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation_vz.png)
+- The "[QEMU](https://www.qemu.org/documentation/)" option is enabled by default and is used to run a guest operating system.
```

Comment:
I'm not familiar with how other pages in this set are formatted, but seeing bullet points used for captions feels a little odd, since they're single items and  not part of a list. If the bullets are intended to make the text line up under the image, could you indent instead?

Reply:
Tbh not sure why I bulleted and agree with your points, removing the formatting.